test/integration/Makefile

```diff
@@ -326,3 +326,6 @@ test_lookup_properties:
 
 args:
 	(cd targets/args && ./runme.sh $(TEST_FLAGS))
+
+test_callback_retry_task_name:
```

Comment:
This won't be run automatically as it's not included in `all`or `other` target. Is this intentional?